lib/core/src/Cardano/Wallet/DB/Transactions/Select.hs

```diff
@@ -0,0 +1,360 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TupleSections #-}
+
+module Cardano.Wallet.DB.Transactions.Select where
+
+
+import Cardano.DB.Sqlite
+    ( chunkSize )
+import Cardano.Wallet.DB.Sqlite.Schema
```

Comment:
Hi @paolino 

Our [coding standards document](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md) specifies to use [explicit imports by default](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#we-use-explicit-imports-by-default-and-favor-qualified-imports-for-ambiguous-functions).

```suggestion
import Cardano.Wallet.DB.Sqlite.Schema
    (add things here)
```

Could you go through this PR and make sure that we are not adding any other implicit imports?

Many thanks!